test/tests/selftest/pbs_default_timeout.py

```diff
@@ -87,6 +84,7 @@ def test_timeout_decorator_greater_default_value(self):
         testcase timeout value consider as timeout decorator value
         """
         try:
-            self.common_sleep_function(MINIMUM_TESTCASE_TIMEOUT+1)
+            time.sleep(MINIMUM_TESTCASE_TIMEOUT+1)
         except TimeOut:
-            raise TimeOut('Test Timeout before complete expected time')
+            mssg = 'Test Timeout before complete expected time'
```

Comment:
actually, there can be a race condition here: if the setUp takes a long time (e.g - if it has to delete a large number of jobs from the previous test), then the time taken by setUp + time.sleep(MINIMUM_TESTCASE_TIMEOUT+1) might actually cross timeout(800). So, instead of failing the test outright, have a assert to check that the timeout was for 800, if not then fail the test.